egs/mgb2/asr/local/mgb_train_lms.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
```

Comment:
We do not have to include this.
Please remove it.

Reply:
done